ush/python/pygfs/task/atm_analysis.py

```diff
@@ -100,9 +103,15 @@ def initialize(self: Analysis) -> None:
         FileHandler(self.get_bkg_dict(AttrDict(self.task_config))).sync()
 
         # generate variational YAML file
-        logger.debug(f"Generate variational YAML file: {self.task_config.jedi_yaml}")
-        save_as_yaml(self.task_config.jedi_config, self.task_config.jedi_yaml)
-        logger.info(f"Wrote variational YAML to: {self.task_config.jedi_yaml}")
+        logger.debug(f"Generate variational YAML file: {self.task_config.var_yaml}")
+        save_as_yaml(self.task_config.jedi_config, self.task_config.var_yaml)
+        logger.info(f"Wrote variational YAML to: {self.task_config.var_yaml}")
+
+        # generate FV3 increment converter YAML file
+        logger.debug(f"Generate FV3 increment converter YAML file: {self.task_config.fv3inc_yaml}")
+        fv3inc_yaml = parse_j2yaml(self.task_config.FV3INCYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
```

Comment:
@aerorahul The problem with changing FV3INCYAML to JEDIYAML in config.atmanlfv3 is that, in the init job here, JEDIYAML is already the variational yaml. It's possible instead that I could stage the FV3 increment converter yaml in the the atmanlfv3inc job itself rather than the atmanlinit job.

Reply:
The problem with that though is I will be using a job dedicated to running an executable to stage a YAML, and I thought the purpose of atmanlinit and atmanlfinal was for things like moving around files.